code/__DEFINES/shuttles.dm

```diff
@@ -21,6 +21,7 @@
 #define SHUTTLE_HEIGHT_TOO_LARGE "height_too_large"
 #define SHUTTLE_ALREADY_DOCKED "we_are_already_docked"
 #define SHUTTLE_SOMEONE_ELSE_DOCKED "someone_else_docked"
+#define SHUTTLE_DOCK_OBSTRUCTED "shuttle_dock_obstructed"
```

Comment:
@coiax 

Added something similar to stop escape shuttles crushing everything I thought. Maybe I removed it.

Also would this block the asteroid shuttle or pod from working then?